draft-ietf-acme-dns-account-01.mkd

```diff
@@ -122,7 +122,7 @@ A client can fulfill this challenge by performing the following steps:
 - Compute the SHA-256 digest {{FIPS180-4}} of the key authorization
 - Construct the validation domain name by prepending the following label to the domain name being validated:
 
-      "_acme-challenge_" || base32(SHA-256(Account Resource URL)[0:9])
+      "_" || base32(SHA-256(Account Resource URL)[0:9]) || "_acme-challenge"
```

Comment:
This should be `"._acme-challenge"` here, right?

Reply:
Yes good point, thank you!